app/config.js

```diff
@@ -75,4 +75,7 @@ module.exports = {
 
   // content and user journey if they can order for multiple organisations
   showProxy: process.env.SHOW_PROXY || 'true',
+
+  // Cookie consent expiration date
+  buyingcatalogueCookieConsentExpiration: 1621594398846,
 };
```

Comment:
This needs to be defined in buying-catalogue-library otherwise you would have to update this variable in every front-end project to expire the cookie.